libs/logging/src/logging.cpp

```diff
@@ -78,7 +78,7 @@ namespace hpx { namespace util {
                 }
                 return hpx::util::logging::level::debug;
             }
-            catch (boost::bad_lexical_cast const&)
+            catch (hpx::util::bad_lexical_cast const&)
```

Comment:
The `try`/`catch` is not necessary anymore, I think.

Reply:
It is still necessary, but it should catch `std::invalid_argument` instead.